components/policy-mgt/org.wso2.carbon.policy.mgt.core/src/main/java/org/wso2/carbon/policy/mgt/core/mgt/impl/PolicyManagerImpl.java

```diff
@@ -726,4 +726,17 @@ public boolean setPolicyApplied(DeviceIdentifier deviceIdentifier) throws Policy
             throw new PolicyManagementException(msg, e);
         }
     }
+
+    @Override
+    public int getPolicyCount() throws PolicyManagementException {
+        int policyCount = 0;
+        try {
+            policyCount = policyDAO.getPolicyCount();
```

Comment:
Redundant variable assignment. You can probably return the output of "policyDAO.getPolicyCount()" directly.